site2/docs/bookkeeper-proposals.md

```diff
@@ -0,0 +1,140 @@
+    1. The final PR for changing BP state will be used as the criteria for marking a BP as completed.
+1. If a BP is failed or rejected:
+    1. Update the PR to change the state of a BP
+        - state: "Discarded"
+        - add a paragraph at the first paragraph of this BP for describing the reasons.
+        - moving the BP from `Inprogress` to `Discarded`.
+    2. Once the PR is updated, committers can merge this proposal PR and close the master issue of this BP.
+
+## All Proposals
+
+This section lists all the _bookkeeper proposals_ made to BookKeeper.
+
+*Next Proposal Number: 41*
+
+### Inprogress
+
+Proposal | State
+:--------|:-----
+[BP-4 - BookKeeper Lifecycle Management](https://cwiki.apache.org/confluence/display/BOOKKEEPER/BP-4+-+BookKeeper+Lifecycle+Management) | Draft
+[BP-8 - Queue based auto rereplicator](https://cwiki.apache.org/confluence/display/BOOKKEEPER/BP-8+-+Queue+based+auto+rereplicator) | Draft
+[BP-12 - Improve documentation](https://cwiki.apache.org/confluence/display/BOOKKEEPER/BP-12+-+Improve+documentation) | Accepted
+[BP-14 Relax durability](https://cwiki.apache.org/confluence/display/BOOKKEEPER/BP-14+Relax+durability) | Accepted
+[BP-16: Thin Client - Remove direct metadata storage access from clients](https://cwiki.apache.org/confluence/display/BOOKKEEPER/BP-16%3A+Thin+Client+-+Remove+direct+metadata+storage+access+from+clients) | Draft
+[BP-18: LedgerType, Flags and StorageHints](https://cwiki.apache.org/confluence/display/BOOKKEEPER/BP-18%3A+LedgerType%2C+Flags+and+StorageHints) | Accepted
+[BP-26: Move distributedlog library as part of bookkeeper](../../bps/BP-26-move-distributedlog-core-library) | Accepted
+[BP-27: New BookKeeper CLI](../../bps/BP-27-new-bookkeeper-cli) | Accepted
+[BP-28: use etcd as metadata store](../../bps/BP-28-etcd-as-metadata-store) | Accepted
+[BP-29: Metadata API module](../../bps/BP-29-metadata-store-api-module) | Accepted
+[BP-30: BookKeeper Table Service](https://docs.google.com/document/d/155xAwWv5IdOitHh1NVMEwCMGgB28M3FyMiQSxEpjE-Y/edit#heading=h.56rbh52koe3f) | Accepted
+[BP-31: BookKeeper Durability Anchor](../../bps/BP-31-durability) | Accepted
+[BP-32: Advisory (optimistic) write close](../../bps/BP-32-advisory-write-close) | Accepted
+[BP-33: Move releasing docker images out of main repo](../../bps/BP-33-building-official-docker-imags) | Draft
+[BP-34: Cluster Metadata Checker](../../bps/BP-34-cluster-metadata-checker) | Accepted
+[BP-35: 128 bits support](../../bps/BP-35-128-bits-support) | Accepted
+[BP-36: Stats documentation annotation](../../bps/BP-36-stats-documentation-annotation) | Accepted
+[BP-37: Improve configuration management for better documentation](../../bps/BP-37-conf-documentation) | Accepted
+[BP-38: Publish Bookie Service Info on Metadata Service](../../bps/BP-38-bookie-endpoint-discovery) | Accepted
+[BP-41: Separate BookieId from Separate BookieId from Bookie Network Address](../../bps/BP-41-bookieid) | Draft
```

Comment:
we may add BP-42?

Reply:
@nicoloboschi IMHO it is better to focus on porting the current website.
BP page is a mess we can fix it later